metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py

```diff
@@ -891,10 +889,7 @@ def _to_metadata_events(  # noqa: C901
 
         mce = MetadataChangeEvent(proposedSnapshot=dataset_snapshot)
         mcp = MetadataChangeProposalWrapper(
-            entityType="dataset",
```

Comment:
Oh this is cool, we guess the type from the urn

Reply:
Yep - I'm trying to remove these where possible since it feels like a bunch of clutter